### ISSUE TITLE
Gegenereerde .NET code veroorzaakt compiler warning door NaamPersoon/Naam constructie

### ISSUE THREAD

**Issue by MelvLee** [VNG-Realisatie/Haal-Centraal-BRP-bevragen#850]
Deze issue is gemeld door @KayodeBakker 

Overerving van de Naam component door de NaamPersoon component veroorzaakt in .NET gegenereerde code de volgende compiler warning: 'NaamPersoon.InOnderzoek' hides inherited member 'Naam.InOnderzoek'.

In standaard situaties wil je bij overerving dat de kind component (in dit geval NaamPersoon) alle properties van het ouder component (in dit geval Naam) krijgt. Als aan het kind component een property (in dit geval InOnderzoek) wordt toegevoegd met dezelfde naam als een parent property, dan is parent property niet meer te bevragen. In dit specifiek geval is dat geen issue omdat het type (NaamPersoonInOnderzoek) van de NaamPersoon.InOnderzoek het type (NaamInOnderzoek) van Naam.InOnderzoek overerft, waardoor er geen informatie verloren gaat.

Een nette oplossing hiervoor is om een NaamBase component te introduceren zonder de InOnderzoek property. Deze kan dan door zowel Naam als door NaamPersoon wordt overerft met hun eigen InOnderzoek property

**Comment by KayodeBakker** [comment]
@fsamwel @MelvLee 
ik zie nu dat er een v2.0 tag erbij is gekomen. Ik neem aan dat de wijziging van Melvin dat naar develop is gegaan, ook naar master komt als onderdeel van v1.3.0?

**Comment by MelvLee** [comment]
> @fsamwel @MelvLee ik zie nu dat er een v2.0 tag erbij is gekomen. Ik neem aan dat de wijziging van Melvin dat naar develop is gegaan, ook naar master komt als onderdeel van v1.3.0?

Ja, ik wacht tot pull request #859 wordt ge-approved. Dan kan ik van de wijzigingen die hiermee te maken hebben een p.r. maken voor de master

**Comment by JohanBoer** [comment]
Volgens mij is dit issue zowel in de master als in develop opgelost

**Comment by KayodeBakker** [comment]
Eens
